### PR TITLE
feat: Add `expand-contractions-en` package

### PR DESCRIPTION
**Pull request by lewisblackburn**
This expands contractions e.g. `aren't` → `are not` . 

This is useful for when writing essays, as professional writing often requires a more formal and precise language style, eliminating contractions contributes to a more polished and academic tone.

### CODE REVIEW

File: packages/expand-contractions-en/0.1.0/README.md
Comment (smeech) [inline packages/expand-contractions-en/0.1.0/README.md:1]: Many duplicate lines - if you sort them in an editor, you can remove them. Sorting them would look better, too!

File: packages/expand-contractions-en/0.1.0/package.yml
Comment (smeech) [inline packages/expand-contractions-en/0.1.0/package.yml:1]: A number of duplicate expansions. If you can sort these, it'll make it easier for users to comment out any they don't want to use.

**Review by smeech — CHANGES_REQUESTED** [review]
A nice, straightforward package of simple expansions.
No `cmd`, `script`, `shell` or malicious code.
Both `README.md` and `package.yml` would benefit from removing duplicates, and, ideally, sorting into alphabetical order. Let me know if you need any help.
Then it will be safe to merge.

File: packages/expand-contractions-en/0.1.0/package.yml
Comment (lewisblackburn) [reply packages/expand-contractions-en/0.1.0/package.yml:1]: Thanks for the feedback! I've made the requested changes:

    Standardized and alphabetized the contraction-to-expansion mappings in the markdown table.
    Removed duplicate entries.
    Added any missing contractions.
    Reformatted the table in the README.md

Let me know if there's anything else you'd like to see updated!

I am going to create a new PR as I can't commit to this one.

**Comment by smeech** [comment]
I presume you're still working on this - just to let you know I am keeping an eye on it.

**Comment by lewisblackburn** [comment]
I think everything should be ready to go now, I'd be happy make any more changes. I apologise if there are any mistakes as I am still learning how to use GitHub 😅

**Comment by smeech** [comment]
I've lost track of the `_manifest.yml` and `README.md` files.

Don't worry - I've learned a lot in the last few months!

**Comment by smeech** [comment]
Thank you - we're getting there!

Would you kindly:
- Rename the package folder to `/0.1.0/expand-contractions-en` as in your original PR, and in `_manifest.yml`, adding a version subfolder
- Add a `homepage:` parameter to `_manifest.yml` even if the value is just something like: https://github.com/lewisblackburn/hub/tree/main/packages/expand-contractions-en (renamed as above), or https://github.com/lewisblackburn/hub
- Change the installation instructions in `README.md` to download from the Hub, i.e. match those in the [docs](https://espanso.org/docs/packages/basics/)

please?

**Comment by lewisblackburn** [comment]
Yes of course no worries, I will do that ASAP. Thanks for your patience.

**Review by smeech — APPROVED** [review]

**Comment by smeech** [comment]
I'm afraid you omitted a period in `/0.1.0/` so it's reading `/0.10/`.

Can you remove the old folder and contents (without the version number), please? While I could delete the files, I might not be able to remove their folder, which may cause problems.

**Comment by lewisblackburn** [comment]
Oh okay sorry about that, one second

**Comment by lewisblackburn** [comment]
Should I add the correct folder back?

**Comment by smeech** [comment]
You may have removed the wrong one. All I can see now in the file-list is the folder and files _without_ the version number.

**Comment by smeech** [comment]
We're now back to two folders, both named `/espanso-contractions-en/0.1.0/`, I'm afraid. We just need one.

**Comment by smeech** [comment]
The folder needs the package name: `expand-contractions-en`, as in `_manifest.yml`. Sorry - I should have spotted that earlier.

**Comment by lewisblackburn** [comment]
I think that should have fixed it

**Comment by smeech** [comment]
Thank you. I've gone ahead with the merge. The package can be found at https://hub.espanso.org/expand-contractions-en. `espanso install expand-contractions-en` works.

Well done!

**Pull request merged**